test/e2e-olm/elasticsearch_test.go

```diff
@@ -512,3 +512,78 @@ func invalidMasterCountTest(t *testing.T) {
 	e2eutil.WaitForDeletion(t, f.Client.Client, cr, cleanupRetryInterval, cleanupTimeout)
 	t.Log("Finished successfully")
 }
+
+func nonDefaultStorageClassTest(t *testing.T) {
```

Comment:
this should be covered by our unit tests, if we specify a storage class name, it should use it

Reply:
The E2E test is removed.